src/api/Feed.js

```diff
@@ -954,6 +978,41 @@ class Feed extends Base {
         });
     }
 
+    /**
+     * Deletes a task collaborator via the API.
+     *
+     * @param {BoxItem} file - The file to which the task is assigned
+     * @param {Task|TaskUpdatePayload} task - The newly deleted or existing task from the API
+     * @param {TaskCollabAssignee} assignee - The user assigned to this task
+     * @param {Function} errorCallback - Task delete error callback
+     * @return {Promise<TaskAssignment>}
+     */
+    deleteTaskCollaborator(
+        file: BoxItem,
+        task: Task | TaskUpdatePayload,
+        assignee: TaskCollabAssignee,
+    ): Promise<TaskCollabAssignee> {
+        if (!file.id) {
+            throw getBadItemError();
+        }
+
+        this.id = file.id;
+        return new Promise((resolve, reject) => {
```

Comment:
async/await

Reply:
API methods like this do not return a Promise, this wraps the callbacks to enable Promise chaining or async/await elsewhere in the component